### PR TITLE
Set DATA_UPLOAD_MAX_MEMORY_SIZE to 1mb

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**

### CODE REVIEW

**Review by iskhakov — APPROVED** [review]

File: engine/settings/base.py [hunk 1]
```diff
@@ -451,7 +451,8 @@
 
 GRAFANA_INCIDENT_STATIC_API_KEY = os.environ.get("GRAFANA_INCIDENT_STATIC_API_KEY", None)
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880
+# Based on 99th percentile of content-length.
```
Comment (iskhakov) [inline engine/settings/base.py:454]: this comment doesn't make sense in oss version code

File: engine/settings/base.py [hunk 2]
```diff
@@ -451,7 +451,7 @@
 
 GRAFANA_INCIDENT_STATIC_API_KEY = os.environ.get("GRAFANA_INCIDENT_STATIC_API_KEY", None)
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880
+DATA_UPLOAD_MAX_MEMORY_SIZE = 1048576
```
Comment (vadimkerr) [inline engine/settings/base.py:454]: Probably worth adding a comment that it's 10MB

**Review by vadimkerr — APPROVED** [review]

File: engine/settings/base.py (hunk 2 above)
Comment (vadimkerr) [reply engine/settings/base.py:454]: Not sure, I thought it's 10MB from the first glance. So probably worth adding a comment that it's 1MB.

Comment (vadimkerr) [reply engine/settings/base.py:454]: Or probably make it even `1 * 1024 * 1024` to be super verbose

Comment (abodacs) [reply engine/settings/base.py:454]: we can add underscore based on PEP 515
so 1048576 will be 1_048_576

**Pull request merged**